src/index.html

```diff
@@ -15,8 +15,97 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header>
+      <div class="container">
+        <a class="header_logo_link">
```

Comment:
Consider adding an `href` attribute to the `<a>` tag for the logo link to make it functional. If it's meant to link to the homepage, you can use `href="/"`.